runtime/IKVM.Runtime/Serialization.cs

```diff
@@ -170,7 +170,10 @@ internal static void AddGetObjectData(TypeBuilder tb)
 				new Type[] { JVM.Import(typeof(SerializationInfo)), JVM.Import(typeof(StreamingContext)) });
 			getObjectData.SetCustomAttribute(securityCriticalAttribute);
 			AttributeHelper.HideFromJava(getObjectData);
+			// AddDeclarativeSecurity does not exist in .net core
```

Comment:
Do we need to do an alternative method in .NET core? Is there a reason it's fine not to handle this in .NET Core?

Reply:
In general we cannot handle something that does not exist.

Reply:
@igorkwwrd Should it throw an exception then so if they make a call that gets to this, they know it's not supported?

Reply:
My goal was to hurt less if possible.  Adding an exception may lead to a compile time error just because something does not exist in the core but it does exist in the framework.  But if you ignore the thing and let it successfully compile, then it works at runtime as it should.

For instance, a user compiles his code for the core and compilation fails saying sorry declarative security is not supported. What the hell, it's the declarative security, it does not exist in the core, in what way should it be supported then?

Reply:
@igorkwwrd Ok, that makes sense. If it's something that just doesn't exist.

Are there any cases where there is something similar but no way to talk to that that you had to skip? Or is everything like this, something that just doesn't exist in Core?